src/typescript/frontend/src/components/pages/home/components/emoji-table/index.tsx

```diff
@@ -13,27 +15,135 @@ import SearchComponent from "./components/Search";
 import FilterOptions from "./components/FilterOptions";
 import { ClientGrid } from "./ClientGrid";
 import type { FetchSortedMarketDataReturn } from "lib/queries/sorting/market-data";
+import { MarketDataSortBy } from "lib/queries/sorting/types";
+import { symbolBytesToEmojis } from "@sdk/emoji_data";
+import { useSearchParams } from "next/navigation";
+import { parseJSON } from "utils";
+import { MARKETS_PER_PAGE } from "lib/queries/sorting/const";
+import useInputStore from "@store/input-store";
 
 export interface EmojiTableProps {
   data: FetchSortedMarketDataReturn["markets"];
   totalNumberOfMarkets: number;
 }
 
+const encodeEmoji = (emojis: string[]) => {
+  const encoder = new TextEncoder();
+  return emojis.reduce(
+    (acc: string, emoji) =>
+      `${acc}${[...encoder.encode(emoji)].map((e) => e.toString(16)).join("")}`,
+    "0x"
+  );
+};
```

Comment:
```suggestion
const encodeEmojis = (emojis: string[]) => {
  const encoder = new TextEncoder();
  return "0x" + emojis.reduce(
    (acc: string, emoji) =>
      `${acc}${[...encoder.encode(emoji)].map((e) => e.toString(16)).join("")}`,
    "",
  );
};
```

Let's put this in the `sdk` where the other emoji data helpers go. I find myself using this a lot as well, it'd be nice to have this in the `sdk` as a helper function.

Also moved the `0x` solely for readability, even though yours was already correct obviously :)